src/symposium-app/symposium-recommendations.html

```diff
@@ -0,0 +1,123 @@
+<link rel="import" href="./symposium-page-card.html">
+<link rel="import" href="../../bower_components/paper-item/paper-item.html">
+<link rel="import" href="../../bower_components/paper-item/paper-item-body.html">
+
+<link rel="import" href="../../bower_components/iron-flex-layout/iron-flex-layout-classes.html">
+
+<dom-module id="symposium-recommendations">
+    <template>
+        <style include="iron-flex iron-flex-alignment"></style>
+        <style is="custom-style">
+            :host {
+                display: block;
+            }
+
+            h1 {
+                margin-top: 0;
+            }
+
+            img {
+                max-width: 100%;
+                max-height: 150px;
+                vertical-align: middle;
+            }
+
+            .recommendations {
+                border-top: 1px rgba(128,128,128,0.3) solid;
+            }
+
+            .company {
+                flex: 1;
+                flex-basis: 150px;
```

Comment:
I would suggest increasing the flex basis to `175px`, this improves the formatting of the text underneath the images.